tor/src/main/resources/osx_64/tor

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
```

Comment:
is this copied from somewhere? If so, can you link the original source in a comment?

Reply:
this is from the tor browser download

Reply:
don't want to edit this file so the shasums don't change, added more context to the readme